crates/compiler/load_internal/tests/test_load.rs

```diff
@@ -1199,3 +1199,64 @@ fn app_missing_package_import() {
         err
     );
 }
+
+#[test]
+fn non_roc_file_extension() {
+    let modules = vec![(
+        "main.md",
+        indoc!(
+            r"
+            # Not a roc file
+            "
+        ),
+    )];
+
+    let expected = indoc!(
+        r"
+        ── EXPECTED ROC FILE in tmp/non_roc_file_extension/main.md ─────────────────────
+
+        I am expecting a roc application file with either `.roc` or no
+        extension. Instead I found a file with extension `.md`"
+    );
+    let color_start = String::from_utf8(vec![27, 91, 51, 54, 109]).unwrap();
+    let color_end = String::from_utf8(vec![27, 91, 48, 109]).unwrap();
```

Comment:
I'm a bit unsure why the two tests I'm adding seem sensitive to the color control character, while other tests, like `nested_module_has_incorrect_name` seem to not be sensitive to those extra chars

Reply:
Hmm, yeah strange, I don't know why either :smile: